src/@hooks/useAddressConfig.ts

```diff
@@ -68,12 +68,14 @@ export function useAddressConfig(): UseAddressConfig {
     )
   }
 
-  const getVerifiedAddressName = (address: string) => {
+  const getVerifiedAddressName: (address: string) => string | undefined = (
+    address
+  ) => {
     const addressKey = Object.keys(verifiedAddresses).find((key) => {
       return key.toLowerCase() === address.toLowerCase()
     })
```

Comment:
hint(non-blocking): just in case you'd like the shorter version, this would be the same as:
```ts
const addressKey = Object.keys(verifiedAddresses).find((key) => key.toLowerCase() === address.toLowerCase())
```
Comes down to personal preference imo